ARGO/src/AiSystem.cpp

```diff
@@ -47,6 +45,42 @@ void AiSystem::update(Entity& t_entity)
 
 void AiSystem::meleeAI(TransformComponent* t_posComp, AiComponent* t_aiComponent, ForceComponent* t_forceComponent)
 {
+	t_aiComponent->setState(AIStates::eWander);
+	Entity* tile = m_levelmanager.findAtPosition(t_posComp->getPos());
+	glm::vec2 seekPosition = glm::vec2(0, 0);
+	if (tile)
+	{
+		FlowFieldComponent* flowFieldComp = static_cast<FlowFieldComponent*>(tile->getComponent(ComponentType::FlowField));
+		if (flowFieldComp && flowFieldComp->getWeight() < Utilities::MAX_FLOW_FIELD_WEIGHT)
+		{
+			if (flowFieldComp->getWeight() == 0)
```

Comment:
this kinda doesnt make sense to have on a seperate line as you are already checking the value above

Reply:
I'm checking it against 2 different values? it's not the exact same check on both lines

Reply:
@CathalRedmond you're a 🐒